### DISCUSSION TITLE
Pagination in Ecommerce demo broken after update to 6.9.1

### DISCUSSION THREAD

**Discussion by justbane** [pimcore/pimcore#8779] [Q&A / Support]
Now getting this error using pagination even demo ecommerce car lists.

error is:
```
An exception has been thrown during the rendering of a template ("Some mandatory parameters are missing ("categoryname", "category") to generate a URL for route "shop-category.en".").
```
in app/Resources/views/includes/pagination.html.twig (line 23)

I saw there was talk of updating using zendframework-bridge from laminas - was something changed that broke this in 6.9.1?

**Comment by fashxp** [comment]
thx for reporting, see https://github.com/pimcore/pimcore/issues/8784

**Reply by justbane** [reply]
thanks @fashxp !!
